client-src/src/pages/counselor/inq-summary/inq-summary.html

```diff
@@ -69,19 +69,20 @@ <h2><b>Reminders: </b></h2>
       </ion-row>
     </ion-item-divider>
     <ion-card-content>
-      <ion-card>
+      <ion-card *ngFor="let followUp of currentInqFollowUps; let i = index;">
           <ion-item-divider color="light">
             <ion-row no-padding>
-              <ion-col><h3><b>Follow Up #1</b></h3></ion-col>
-              <ion-col><h3><b>Done On: 30/12/2017 @ 3 PM</b></h3></ion-col>
-              <ion-col><h3><b>Course Offered: AAIP</b></h3></ion-col>
+              <ion-col><h3><b>Follow Up #{{currentInqFollowUps.length-i}}</b></h3></ion-col>
+              <ion-col><h3><b>Done On: {{followUp?.followUpDate | date : "dd/MM/yyyy"}}</b></h3></ion-col>
+              <ion-col><h3><b>Course Offered: N/A</b></h3></ion-col>
             </ion-row>
           </ion-item-divider>
           <ion-card-content>
             <br>
-            <p><b>State:</b>&nbsp;Lorem ipsum dolor sit amet</p>
-            <p><b>Remark:</b>&nbsp;Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-            <p><b>Next Follow Up On:</b>&nbsp;10/01/2018 @ 3 PM</p>
+            <p><b>Status:</b>&nbsp;{{followUp?.followUpStatus}}</p>
+            <p><b>Sub Status:</b>&nbsp;{{followUp?.subStatus}}</p>
+            <p><b>Remark:</b>&nbsp;{{followUp?.remark}}</p>
```

Comment:
Is remark inside of the follow up loop?

Reply:
Yes.

Reply:
Reminder as of now is inquiry level. Not on the level of follow-up

Reply:
This is the remark for the follow up, how is this relevant to the reminder, i am a bit confused.

Reply:
Ok. I got confused